src/discordtipbot/statistics.cr

```diff
@@ -0,0 +1,43 @@
+struct Statistics
+  DB.mapping({
```

Comment:
This kind of takes up a lot of space for not much gain. I would just prefer:
```cr
DB.mapping(
  transactions: {type: Int64, key: "transaction_count"},
  total: {type: BigDecimal, key: "transaction_sum"},
  tips: {type: BigDecimal, key: "tip_sum"},
  soaks: {type: BigDecimal, key: "soak_sum"},
  rains: {type: BigDecimal, key: "rain_sum"}
)
```

Reply:
Done